staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go

```diff
@@ -282,6 +290,7 @@ const percentErrMsg string = "a valid percent string must be a numeric string fo
 
 var percentRegexp = regexp.MustCompile("^" + percentFmt + "$")
 
+// IsValidPercent checks that a string conforms is a valid percent
```

Comment:
I'm not sure this message and the one below are grammatically correct.

Reply:
Me too... waiting for a native speaker.

Reply:
Drop `conforms`?

Reply:
IsValidPercent checks that a string is in the form of a percentage

Reply:
@cheftako @ash2k Thank you